zio-http/src/main/scala/zio/http/socket/WebSocketChannel.scala

```diff
@@ -0,0 +1,50 @@
+package zio.http.socket
+
+import zio._
+
+import zio.http.ChannelEvent.{ChannelRead, ChannelRegistered, ChannelUnregistered, ExceptionCaught, UserEventTriggered}
+import zio.http._
+import zio.http.netty.NettyChannel
+import zio.http.socket.WebSocketChannelEvent
+
+import io.netty.buffer.{ByteBufUtil, Unpooled}
+import io.netty.handler.codec.http.websocketx.{WebSocketFrame => JWebSocketFrame, _}
+
+object WebSocketChannel {
```

Comment:
It is odd for this companion object to appear in the public API, even though it has no public methods.